src/generators/src/packagejson.js

```diff
@@ -72,5 +90,9 @@ module.exports =
       }
     })
 
+    await childProcess(`npm install`)
+
     process.env['NODE_MODULES'] = `${__dirname}/../../../node_modules`
+    // eslint-disable-next-line no-console
+    console.info(`ok`)
```

Comment:
Why this console.info here?

Reply:
I followed the pattern that appears at the end of other steps in project generation

Reply:
Could you please remove those?

Reply:
Sure

Reply:
`console.info` removed